### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by BrittanyBlake**
In this project, I have implemented the following features: 
- On the page load, it will show the data of the default location (Granada, Spain)
- The user can search for data on different cities
- The user can toggle between Celsius and Fahrenheit
- The background images change to match the weather. 

[Live Demo Link](https://weatherapp-brittany.netlify.app/)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+const footerDesc = document.querySelector('.currentTempDesc');
+const footerMax = document.querySelector('.TodayTemp');
+const footerMin = document.querySelector('.TonightTemp');
+const tomorrowTemp = document.querySelector('.TomorrowTemp');
+
+const key = '929f44f39fa3d7465cb2e466b81dfc25';
+
+const toFahrenheit = (temp) => temp * (9 / 5) + 32;
+const convertTemp = (data) => {
+  const mainTempF = toFahrenheit(data.main.temp);
```
Comment (Forison) [inline src/index.js]: * [x] Kindly use object destructuring to avoid repeting your self. For instance
```const {main} = data ```

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Change required
# 🙋‍♂️ Brittany
#### You have done a great job on this project. Your files were well structured the code was very clean, modular, and very legible. Your application's UI was amazing. 👏👏👏!!!
Let us make the project perfect by addressing these issues.
* [x]  Let us find a way to display to the user that, a given location does not exist when they enter a wrong city name or country. You can use ```try and catch``` or rely on the HTTP response.
* [x] Let us use object destructuring to get the values from an object. This way you can avoid repeating a lot of code. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/destructuring_requirement.png) from Pathwright)
* [x] The necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
- KIndly add `"dev": "webpack --mode development",` to the script in `package.json`

**[OPTIONAL]**

* [x] Please make it a habit to always add a description to the repo you create on Github, it makes it look more professional.

### That is it kindly make the change and submit a re-review🤗.

#### Happy coding
Addo Forison

**Comment by BrittanyBlake** [comment]
Thank you the review. I have implemented all of the changes requested by the TSE.

**Review by bellom — APPROVED** [review]
## Hi @BrittanyBlake  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**